corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -114,31 +161,28 @@ def _forms_per_user_per_month(self):
             )
         )
 
-        def _format_rows(query_):
-            return [
-                (row['month'].isoformat(), row['num_users'], row['avg_forms'], row['std_dev'])
-                for row in query_
-            ]
+        total_users = 0
+        total_average_forms = 0
+        months = 0
 
-        headers = ['Month', 'Active Users', 'Average forms per user', 'Std Dev']
-        self.stdout.write('All user stats')
-        self._print_table(
-            headers,
-            _format_rows(
-                user_stat_from_malt
-                .filter(user_type__in=['CommCareUser'])
-                .filter(user_id__in=self.active_not_deleted_users)
-                .filter(num_of_forms__gte=performance_threshold)
-            )
-        )
+        for stat in user_stat_from_malt:
+            total_average_forms += stat['avg_forms']
+            total_users += stat['num_users']
+            months += 1
 
-        self.stdout.write('System user stats')
-        self._print_table(
-            headers,
-            _format_rows(user_stat_from_malt.filter(username='system'))
-        )
+        resource_model['total_users'] = total_users
+
+        if months > 0:
+            resource_model['monthly_forms_per_user'] = total_average_forms/months
+
+        monthly_user_stats = user_stat_from_malt\
+            .filter(user_type__in=['CommCareUser'])\
+            .filter(user_id__in=self.active_not_deleted_users)\
+            .filter(num_of_forms__gte=performance_threshold)
+
```

Comment:
nit, if you surround this in `()` you wouldn't need the `\`

```suggestion
        monthly_user_stats = (user_stat_from_malt
            .filter(user_type__in=['CommCareUser'])
            .filter(user_id__in=self.active_not_deleted_users)
            .filter(num_of_forms__gte=performance_threshold))

```

Reply:
+1